ActiveLabelTests/IntegrationTests/PhoneParsingIntegrationTests.swift

```diff
@@ -9,7 +9,7 @@
 import XCTest
 @testable import ActiveLabel
 
-final class PhoneParsingTests: BaseTestCase {
+final class PhoneParsingIntegrationTests: BaseIntegrationTestCase {
```

Comment:
BaseTestCase vs BaseIntegrationTestCase?

Reply:
BaseTestCase was actually settings things up for integration testing rather then unit testing, hence the name change.